content/techniques/task-scheduling.md

```diff
@@ -225,7 +225,7 @@ The `@nestjs/schedule` module provides a dynamic API that enables managing decla
 Obtain a reference to a `CronJob` instance by name from anywhere in your code using the `SchedulerRegistry` API. First, inject `SchedulerRegistry` using standard constructor injection:
 
 ```typescript
-constructor(private schedulerRegistry: SchedulerRegistry) {}
+constructor(private readonly schedulerRegistry: SchedulerRegistry) {}
```

Comment:
we usually tend to not use `readonly` in the docs to not overcomplicate examples (make them as short as we can)